docs/json.md

```diff
@@ -4,6 +4,12 @@ Adding support for JSON (or other format) bodies in requests/responses is a matt
 
 Each integration is available as an import, which brings the implicit `BodySerializer`s and `asJson` methods into scope. Alternatively, these values are grouped intro traits (e.g. `sttp.client.circe.SttpCirceApi`), which can be extended to group multiple integrations in one object, and thus reduce the number of necessary imports.
 
+Following data class will be used through the next few examples:
+```scala mdoc
+case class Payload(data: String)
```

Comment:
Maybe we could make this more symmetric, e.g. `RequestPayload` and `ResponsePayload`?